<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SongWords</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/songwords.png</normaloff>:/images/songwords.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QListView" name="songlistv">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="contente">
       <property name="centerOnScroll">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="newsongb">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadb">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/document-open.svg</normaloff>:/images/document-open.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveb">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="document-save"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gob">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loopc">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/images/AnotherLooper.svg</normaloff>:/images/AnotherLooper.svg</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bpm:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tempoe">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>1.212</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="namee">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Girl</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomInB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-in"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOutB">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="zoom-out"/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>newsongb</sender>
   <signal>clicked()</signal>
   <receiver>actionNew</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>24</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
